dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -81,7 +81,9 @@
             <!-- oaire:citation* -->
             <xsl:apply-templates
                 select="doc:metadata/doc:element[@name='oaire']/doc:element[@name='citation']" mode="oaire"/>
-
+			<!-- CREATIVE COMMON LICENSE -->
+			<xsl:apply-templates
+				select="doc:metadata/doc:element[@name='others']/doc:field[@name='cc']" mode="oaire" />
```

Comment:
I think you could have this with a different structure like a specific element with multiple fields

Reply:
done